src/nodedef.cpp

```diff
@@ -792,9 +792,16 @@ class CNodeDefManager: public IWritableNodeDefManager {
 	virtual void resetNodeResolveState();
 	virtual void mapNodeboxConnections();
 	virtual bool nodeboxConnects(MapNode from, MapNode to, u8 connect_face);
+	virtual aabb3f getSelectionBoxUnion();
+	virtual core::aabbox3d<s16> getSelectionBoxIntUnion();
```

Comment:
I don't think this is important. Returning by value is safer, and it only costs one aabbox copy per one call of `getPointedThing()`.